test/distribution/test_xmllib.py

```diff
@@ -0,0 +1,22 @@
+from dsp_tools.xmllib.models.dsp_base_resources import AnnotationResource
+from dsp_tools.xmllib.models.resource import Resource
+from dsp_tools.xmllib.models.root import XMLRoot
+from dsp_tools.xmllib.value_checkers import is_bool_like
+from dsp_tools.xmllib.value_converters import convert_to_bool_string
+
+
+def test_xmllib() -> None:
+    """
+    This does NOT test the xmllib.
+    This only tests if all dependencies used in the xmllib are shipped in the distribution.
+    For this purpose, it is enough to import all modules of the xmllib.
+    Indirect imports are okay, too, i.e. modules that are imported by modules that are imported here.
+    """
+    root = XMLRoot("0000", "my_onto")
+    my_res = Resource.new("my_res", "restype", "label")
+    if is_bool_like("0"):
+        my_res.add_bool(convert_to_bool_string("0"), "my_prop")
+    my_res.add_file("my_file")
+    annotation = AnnotationResource.new("res_id", "label", "annotation_of", ["comment1", "comment2"])
+    root.add_resource(annotation)  # type: ignore[arg-type]
+    root.add_resource(my_res)
```

Comment:
Not sure how smart this test is. It's just the only idea that I had how to solve the problem.

Reply:
It is better than nothing. We will have to take care to test it better later. Perhaps we can take some of noemi's scripts for that